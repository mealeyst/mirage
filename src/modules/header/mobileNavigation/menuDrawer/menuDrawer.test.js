import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { MenuDrawer } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) MenuDrawer', () => {
  const createMenuDrawer = (props) => {
    return shallowWithTheme(<MenuDrawer {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createMenuDrawer())
    .toMatchSnapshot()
  })
})
