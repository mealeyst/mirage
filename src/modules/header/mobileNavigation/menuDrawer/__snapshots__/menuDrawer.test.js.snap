// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) MenuDrawer matching the snapshot 1`] = `
.c0 {
  position: fixed;
  top: 0;
  left: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  height: 100vh;
  z-index: 4;
  -webkit-transform: translateX(-200%);
  -ms-transform: translateX(-200%);
  transform: translateX(-200%);
  -webkit-transition: -webkit-transform 0.6s ease-in-out;
  -webkit-transition: transform 0.6s ease-in-out;
  transition: transform 0.6s ease-in-out;
}

.c0.open {
  -webkit-transform: translateX(0%);
  -ms-transform: translateX(0%);
  transform: translateX(0%);
  -webkit-transition: -webkit-transform 0.6s ease-in-out;
  -webkit-transition: transform 0.6s ease-in-out;
  transition: transform 0.6s ease-in-out;
}

.c0.open .overlay {
  background-color: rgba(255,255,255,0.6);
  -webkit-animation: jpGFJL 0.85s ease-in-out;
  animation: jpGFJL 0.85s ease-in-out;
}

.c0 nav {
  padding: 0;
  padding-bottom: 80px;
  list-style: none;
  border-top: 1px solid #0073D1;
  border-right: 1px solid #0073D1;
  border-left: 1px solid #0073D1;
  background-color: #FFFFFF;
  width: 75%;
  height: 100%;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  overflow-x: scroll;
}

.c0 ul {
  width: 100%;
  box-sizing: border-box;
}

.c0 .overlay {
  background-color: rgba(255,255,255,0.0);
  -webkit-animation: hwFNVQ 0.1s ease-in-out;
  animation: hwFNVQ 0.1s ease-in-out;
  width: 25%;
}

.c0 .c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  -webkit-align-self: flex-end;
  -ms-flex-item-align: end;
  align-self: flex-end;
  margin-top: 1.75rem;
  margin-right: 1.5rem;
  margin-left: auto;
}

<BaseMenuDrawer
  className="c0"
  open={false}
  position="fixed"
/>
`;
